SCons/Tool/msvc.xml

```diff
@@ -438,13 +453,13 @@ Valid values for Windows are
 <literal>x86</literal>,
 <literal>arm</literal>,
 <literal>i386</literal>
-(for 32 bits); 
+(for 32 bit);
 <literal>amd64</literal>,
 <literal>arm64</literal>,
 <literal>emt64</literal>,
 <literal>x86_64</literal>
-(for 64 bits);
-and <literal>ia64</literal> (Itanium).
+(for 64 bit);
+and <literal>ia64</literal> (Itanium) (deprecated).
```

Comment:
Why add deprecated?
SCons will support that as long as we support a MSVC compiler toolchain which can target itanium?